auslib/db.py

```diff
@@ -83,6 +83,34 @@ class ChangeScheduledError(SQLAlchemyError):
     for data consistency reasons."""
 
 
+class JSONColumn(sqlalchemy.types.TypeDecorator):
```

Comment:
Maybe some comments around these two classes. The functions they contain are opaque until you go look them up.